src/makeRobotAccountant.js

```diff
@@ -6,7 +6,37 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callsCount = 0;
+
+  return (a) => (b) => {
```

Comment:
let's better put inside functions on new line to improve code readability
```suggestion
  return (a) => {
    return (b) => {
```